sos/policies/distros/__init__.py

```diff
@@ -200,6 +203,11 @@ def init_kernel_modules(self):
             if config_strings[builtin] in kconfigs:
                 self.kernel_mods.append(builtin)
 
+    def join_sysroot(self, path):
```

Comment:
Why we can't use `archive`'s `join_sysroot` directly?

Reply:
1. `Policy` does not have access to an `Archive()`  directly (but `Archive()` _does_ have access to `Policy()`)
2. This is being used for kmod detection using the local files - `Archive()` isn't in the equation for accessing these. 
3. This commit has `Policy` be the ultimate source of `sysroot` truth, so it'd be a bit weird to wire up `Archive` here when `Archive` is dependent on `Policy` to know what `sysroot` even is.

Reply:
Mea culpa, I should think before I write - esp. at Friday's evening :)